sdk/spring/azure-spring-boot-test-keyvault/src/test/java/com/azure/test/keyvault/KeyVaultIT.java

```diff
@@ -131,30 +127,25 @@ public void keyVaultWithAppServiceMSI() {
     }
 
     @Test
-    @Ignore
-    public void keyVaultWithVirtualMachineMSI() throws Exception {
+    public void keyVaultWithVirtualMachineMSI() {
         final VirtualMachine vm = AZURE.virtualMachines().getByResourceGroup(SPRING_RESOURCE_GROUP, VM_NAME);
 
         final String host = vm.getPrimaryPublicIPAddress().ipAddress();
 
-        // Upload app.jar to virtual machine
-        final MavenBasedProject app = new MavenBasedProject("../azure-spring-boot-test-application");
-        app.packageUp();
-
-        final File file = new File(app.artifact());
-
-        if (!file.exists()) {
-            throw new FileNotFoundException("There's no app.jar file found.");
-        }
-        try (SSHShell sshShell = SSHShell.open(host, 22, VM_USER_USERNAME, VM_USER_PASSWORD);
-            FileInputStream fis = new FileInputStream(file)) {
-            LOGGER.info("Uploading jar file...");
-            sshShell.upload(fis, "app.jar", "", true, "4095");
-        }
-
-        // run java application
         final List<String> commands = new ArrayList<>();
         commands.add(String.format("cd /home/%s", VM_USER_USERNAME));
+        commands.add("apt-get install maven -y");
+        commands.add("apt-get install git");
```

Comment:
Please install `java maven` in this file: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-test-keyvault/install_java.sh